src/test/java/uk/gov/justice/hmpps/prison/service/BookingServiceTest.java

```diff
@@ -1655,4 +1668,114 @@ void noVisits() {
             assertThat(summary.getHasVisits()).isFalse();
         }
     }
+
+    @Nested
+    class addPersonalCareNeed{
```

Comment:
not sure it matters but linters/IntelliJ might complain by lowercase class name.
I would tend to do this:

`@DisplayName("addPersoanlCareNeed")
class AddPersoanlCareNeed {`

Reply:
fix class name